src/ui/public/index_patterns/route_setup/load_default.js

```diff
@@ -49,8 +53,22 @@ export default function (opts) {
 
         // Avoid being hostile to new users who don't have an index pattern setup yet
         // give them a friendly info message instead of a terse error message
-        createFirstIndexPatternPrompt.show();
-        setTimeout(createFirstIndexPatternPrompt.hide, 15000);
+        topBanners.set({
```

Comment:
How about we make `topBanners` responsible for assigning an ID, which will be returned from `set`? This way we don't need to worry about collisions. This takes some of the burden of the consumer of worrying about collisions, especially since we don't throw an error in the case of a collision.

Reply:
The `id` has become optional and it returns the `id` that it used (whether specified or generated). I also switched the documentation to suggest against doing it and changed the examples to show not using it.